src/screens/SettingsScreen.js

```diff
@@ -105,7 +111,7 @@ export const SettingsScreen = () => {
 
       // settings should sometimes contain matomo analytics next, depending on server settings
       if (settings.matomo) {
-        const { consent: matomoValue = false } = await storageHelper.matomoSettings();
+        const { consent: matomoValue } = await matomoSettings();
```

Comment:
nice. we can get rid of default value because of

```
// if there are no matomo settings yet, init without consent
if (!settings) {
  settings = {
    consent: false
  };
}
```

right?

Reply:
correct. we always initialize it as part of the helper function